internal/workflow/processing.go

```diff
@@ -368,9 +368,8 @@ func (w *ProcessingWorkflow) SessionHandler(sessCtx temporalsdk_workflow.Context
 		}).Get(activityOpts, nil)
 	}
 
-	// Stop here for the the Archivematica workflow. AIP creation, review, and
-	// storage are handled entirely by Archivematica and the AM Storage Service.
-	if w.preservationTaskQueue == temporal.AmWorkerTaskQueue {
+	// Stop here for the Archivematica workflow.
+	if w.taskQueue == temporal.AmWorkerTaskQueue {
 		return nil
 	}
```

Comment:
I'm not sure if I'm understanding this correctly, but it seems we are assuming that taskQueue will be either "am" or "a3m". This assumption might inadvertently prevent multiple instances of Enduro from sharing the same Temporal namespace, as other deployments could have workers on the same queue. Perhaps this isn't an issue, since namespaces should not be shared, but I wanted to raise this point in case it could be a concern. Just like you wouldn't use the same MySQL database for two different apps.

Reply:
Yes, we are assuming taskQueue will be either "am" or "a3m".  

I guess the alternative would be to have a cfg variable for the name of each task queue, e.g. `cfg.AMTaskQueue` and `cfg.A3mTaskQueue`?  The down side I see to this approach is config variable proliferation - in addition to the these two config vars, we need to add a new config variable every time we add another preservation worker with yet another task queue.  Maybe the chance of adding a new preservation worker is small enough that this isn't a real problem.

Are there other alternatives I'm missing?

Reply:
Yeah that'd be it, but agree not a deal breaker.